svf-llvm/lib/LLVMUtil.cpp

```diff
@@ -1042,6 +1044,11 @@ const std::string SVFValue::toString() const
     {
         const Value* val =
             LLVMModuleSet::getLLVMModuleSet()->getLLVMValue(this);
+        if (val == nullptr)
+        {
```

Comment:
assert(isa<SVFCallInst>(this) && "this is not a manually created call inst by ExtAPI");